pkg/server/event_handler.go

```diff
@@ -266,11 +266,21 @@ func (h *EventHandler[T, P]) Handle(ctx context.Context, m pubsub.Message) error
 
 func (h *EventHandler[T, P]) generatePmapEventBytes(ctx context.Context, m pubsub.Message) ([]byte, error) {
 	// Get the GCS object as a proto message given GCS notification information.
-	p, err := h.getGCSObjectProto(ctx, m.Attributes)
```

Comment:
Do we need include get gcs object error as part of the processErr, I thought the process error is just for processors?

Reply:
My bad, removed.

Reply:
Can you move it to somewhere near line 283?

Reply:
Moved to line 282.